src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+---
+title: Issues API using Nerdgraph
+metaDescription: "Set capabilities and entitlements in Detection and AI"
```

Comment:
check out names of other docs. Note that we don't use 'API' to refer to specific aspects of using NerdGraph; we'd just refer to something like 'Use NerdGraph to create and configure issues' (or whatever). Title might be something like "NerdGraph tutorial: Configure alerting and applied intelligence issues"